src/Analyzers/CSharp/CodeFixes/MakeLocalFunctionStatic/PassInCapturedVariablesAsArgumentsCodeFixProvider.cs

```diff
@@ -30,39 +29,14 @@ internal sealed class PassInCapturedVariablesAsArgumentsCodeFixProvider() : Synt
 
     public override Task RegisterCodeFixesAsync(CodeFixContext context)
     {
-        var diagnostic = context.Diagnostics.First();
```

Comment:
this pattern was notneeded.  It's exactly what the base SyntaxEditorBasedCodeFixProvider already does for you.